plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts

```diff
@@ -189,9 +189,9 @@ describe('publish:bitbucket', () => {
       remoteUrl: 'https://bitbucket.org/workspace/cloneurl',
       auth: { username: 'x-token-auth', password: 'tokenlols' },
       logger: mockContext.logger,
-      defaultBranch: 'main',
+      defaultBranch: 'master',
```

Comment:
I would strongly vouche to keep `main` where possbile. We are only historically stuck with `master` on this repository. In new repositories we are also using `main` over `master`.

Reply:
This is a consequence of modifying the examples to not longer specify a custom branch for **every** example. By defauly the bitbucket scaffolder module sets the default branch to master if one if not specified.